### PR TITLE
Bug 1155421 - Change 'Discard' string to 'Delete Draft' sms R=aron.bordin

### PR DESCRIPTION
**Pull request by nrbrd**
Change the string 'Discard' -> 'Delete Draft' in the sms app.

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=98f62d917ae9de77e77e8060f17d608e3f1a87b8

**Comment by julienw** [comment]
As you can see in the test results above (direct link is https://treeherder.mozilla.org/logviewer.html#?job_id=905565&repo=gaia) you'll need to update the unit tests as well.

It's a lot easier if you run the unit tests locally. You can read simple instructions in your README: https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/README.md#running-unit-tests

File: apps/sms/locales/sms.en-US.properties [hunk 1]
```diff
@@ -133,6 +133,7 @@ thread-participant-separator = ,\u20
 replace-draft               = Replace existing Draft
 save-as-draft               = Save as Draft
 discard-message             = Discard
```
Comment (julienw) [inline apps/sms/locales/sms.en-US.properties]: you can remove the existing l10nId because we don't use it anymore :)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a430f73a0c1fba44132c70f2a87c918f5731e201

**Comment by julienw** [comment]
I see a marionette test fails too. You need to change also this line: https://github.com/mozilla-b2g/gaia/blob/a69de6f18a940064f831a1c282c697965a7ac8fe/apps/sms/test/marionette/share_activity_test.js#L50

Running these tests manually is more difficult than for unit tests, but you can find information on MDN: https://developer.mozilla.org/en-US/Firefox_OS/Automated_testing/Gaia_integration_tests

Please come on IRC if you have any issue with these tests !

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e37af7f0ab55d73b8dfc87f9a836b4a10c2c1374

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f9a6a581e1689cc15242198b75eb0b4242a3959e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4bbe7c6ca1531208fdaac2cf94de152ec4f84ecb

**Pull request merged**